CMakeLists.txt

```diff
@@ -1,135 +1,176 @@
-# build options and optional modules:
-option(SQLITE_ENABLE_COLUMN_METADATA "enables column metadata"                         OFF)
-option(SQLITE_ENABLE_DBSTAT_VTAB     "enables dbstat virtual table"                    OFF)
-option(SQLITE_ENABLE_FTS3            "enables full text searches version 3"            OFF)
-option(SQLITE_ENABLE_FTS4            "enables full text searches version 3 & 4"        OFF)
-option(SQLITE_ENABLE_FTS5            "enables full text searches version 5"            OFF)
-option(SQLITE_ENABLE_GEOPOLY         "enables Geopoly extention"                       OFF)
-option(SQLITE_ENABLE_ICU             "enables international components for unicode"    OFF)
-option(SQLITE_ENABLE_MATH_FUNCTIONS  "enables the built-in SQL math functions"         ON)
-option(SQLITE_ENABLE_RBU             "enables resumable bulk update extension"         OFF)
-option(SQLITE_ENABLE_RTREE           "enables R*TRee index extension"                  OFF)
-option(SQLITE_ENABLE_STAT4           "enhances query planner under certain situations" OFF)
-option(SQLITE_OMIT_DECLTYPE          "omit declared type of columns"                   ON)
-option(SQLITE_OMIT_JSON              "disables JSON SQL functions"                     OFF)
-option(SQLITE_RECOMMENDED_OPTIONS    "compile by SQLite3 recommended options"          ON)
-option(SQLITE_USE_URI                "enables the default URI filename processing"     OFF)
+# build options and optional modules (https://www.sqlite.org/compile.html):
+option(SQLITE_ENABLE_COLUMN_METADATA	"enables column metadata"				OFF)
+option(SQLITE_ENABLE_DBSTAT_VTAB	"enables dbstat virtual table"				OFF)
+option(SQLITE_ENABLE_FTS3		"enables full text searches version 3"			OFF)
+option(SQLITE_ENABLE_FTS4		"enables full text searches version 3 & 4"		OFF)
+option(SQLITE_ENABLE_FTS5		"enables full text searches version 5"			OFF)
+option(SQLITE_ENABLE_GEOPOLY		"enables Geopoly extention"				OFF)
+option(SQLITE_ENABLE_ICU		"enables international components for unicode"		OFF)
+option(SQLITE_ENABLE_MATH_FUNCTIONS	"enables the built-in SQL math functions"		ON)
+option(SQLITE_ENABLE_RBU		"enables resumable bulk update extension"		OFF)
+option(SQLITE_ENABLE_RTREE		"enables R*TRee index extension"			OFF)
+option(SQLITE_ENABLE_STAT4		"enhances query planner under certain situations"	OFF)
+option(SQLITE_OMIT_DECLTYPE		"omit declared type of columns"				ON)
+option(SQLITE_OMIT_JSON			"disables JSON SQL functions"				OFF)
+option(SQLITE_RECOMMENDED_OPTIONS	"compile by SQLite3 recommended options"		ON)
+option(SQLITE_USE_URI			"enables the default URI filename processing"		OFF)
+option(SQLITE_HAVE_ZLIB			"link against the zlib compression library"		OFF)
+option(SQLITE_WIN32_MALLOC		"use Windows Heap API functions for memory allocation"	OFF)
+option(SQLITE_WIN32_HEAP_CREATE		"forces the Win32 native memory allocator"		OFF)
+option(SQLITE_THREADSAFE		"thread-safety logic in SQLite omitted"			ON)
```

Comment:
Does this change the default to omit thread safety? If so that is quite a breaking change.

Reply:
Hi @rhuijben, Thx for merging!
I've only thrown up compile option to `CMakeLists.txt`
see https://www.sqlite.org/compile.html#threadsafe for details